bitcoind-rpc-test/src/test/scala/org/bitcoins/rpc/v18/PsbtRpcTest.scala

```diff
@@ -130,10 +130,21 @@ class PsbtRpcTest extends BitcoindRpcTest {
     val nodeF = startF.flatMap(_ => client.getNodeAddresses())
 
     //Todo figure out how to check time and port
-    nodeF.map({ result =>
+    nodeF.map { result =>
       assert(result.head.address.isAbsolute)
       assert(result.head.services >= 0)
       assert(result.nonEmpty)
-    })
+    }
+  }
+
+  it should "return active rpc commands" in {
+    val startF = client.start()
+    val rpcinfoF = startF.flatMap(_ => client.getRpcInfo())
+
+    rpcinfoF.map { result =>
+      assert(
+        result.active_commands.head.duration == result.active_commands.head.duration.toMicros) //want to check that it computes to micros correctly

```

Comment:
This test is kind of moot, as this tests the behavior of `FiniteDuration, which is a standard library method. I think this is a better solution: 

```scala
it should "return active rpc commands" in {
    val startF = client.start()

    // kick off a future that shouldn't complete until after getRpcInfo
    startF.flatMap(_ =>
      client.getNewAddress.flatMap(client.generateToAddress(50000, _)))

    val rpcinfoF = startF.flatMap(_ => client.getRpcInfo())

    rpcinfoF.map { result =>
      assert(result.active_commands.length == 2)
    }
  }
```